eth_crowdfund_be/eth_crowdfund_api/migrations/versions/23e7fe5df48f_.py

```diff
@@ -0,0 +1,59 @@
+"""empty message
+
+Revision ID: 23e7fe5df48f
+Revises: 
+Create Date: 2020-09-03 15:51:58.781746
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '23e7fe5df48f'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('campaigns',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('description', sa.Text(), nullable=True),
+    sa.Column('image', sa.String(), nullable=True),
+    sa.Column('manager', sa.String(), nullable=False),
+    sa.Column('contributors', sa.Integer(), nullable=True),
+    sa.Column('upvote', sa.Integer(), nullable=True),
+    sa.Column('min_contribution', sa.Float(), nullable=False),
+    sa.Column('address', sa.String(), nullable=False),
```

Comment:
Is there a way to add `unique = true` on this attribute? I'm sure its something we can add on later as well.

Reply:
The address is required to be unique via line 35 `sa.UniqueConstraint('address')`

Reply:
Nice, I didn't catch that either.